File: PSET1/mario.c
#include<cs50.h>
#include<stdio.h>
 void height(int h)
    {
    if(h<1)
    {
        printf("Height: %i\n",h);
    }
    else if(h>=24)
    {
        printf("Height: %i\n",h);
        printf("please re-enter the height");
        h=get_int();
        height(h);
    }
    else
    {
       printf("Height: %i\n",h);
       for(int i=2;i<=h+1;i++)
       {
           for(int k=h+1-i;k>0;k--)
           {
               printf(" ");
           }
           for(int j=1;j<=i;j++)
           {
               printf("#");
           }
           printf("\n");
       }
    }
}
int main(void)
{
    int h=get_int();
    height(h);
}

File: PSET6/mario.py
import cs50
def main():
    h=cs50.get_int()
    height(h)
#print("name: ", end="")
    #name = cs50.get_string()

def height(h):
    if h < 1:
        print("Height: {}".format(h))
        print("please re-enter the height")
        h=cs50.get_int();
        height(h)
    elif h >= 24:
        print("Height:{}".format(h))
        print("please re-enter the height")
        h=cs50.get_int();
        height(h)
    else:
       print("Height:{}".format(h))
       for i in range(2,h+2):
           for k in range(h+2-i,0,-1):
               print(" ",end="")
           for j in range(1,i+1):
               print("#",end="")
           print()

if __name__ == "__main__":
    main()
